doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,195 @@
+// Custom scripts for the Spyder docs site
+
+
+;(function () {
+
+    'use strict';
+
+    /* Top-level variables */
+
+    // Interactive tour driver options
+    var quickstartDriverOptions = {
+        animate: false,
+        opacity: 0.1,
+        padding: 0,
+        allowClose: false,
+        nextBtnText: 'Next <span class="tour-arrow">→</span>',
+        prevBtnText: '<span class="tour-arrow">←</span> Previous',
```

Comment:
Other suggestions for these arrows:

```suggestion
        nextBtnText: 'Next <span class="tour-arrow">⮞</span>',
        prevBtnText: '<span class="tour-arrow">⮜</span> Previous',
```

Reply:
```suggestion
        nextBtnText: 'Next <span class="tour-arrow">⯈</span>',
        prevBtnText: '<span class="tour-arrow">⯇</span> Previous',
```

Reply:
We already tried that, but we could not get them working consistently across browsers and platforms due to differing font and Unciode support. The arrows that worked on @juanis2112 's machine either displayed inconsistent with one another or didn't work at all on my machine, and the same with via versa, except for the most basic ones, so we reverted to that.

Reply:
Ok, too bad.

Reply:
Also, because of how we're changing the size and height of the arrows and how they render on different systems, they actually line up at different vertical positions depending on OS, browser and font, so we can't just hardcode a padding to cheat it where we want it. I couldn't get it to work consistently so we might just have to set the arrows to the same size as the rest of the text, or get rid of them entirely, to avoid them looking uncontrollably weird, unless @dalthviz has any ideas.